plugins/ua_network_tcp.c

```diff
@@ -42,13 +42,16 @@
 # ifdef __QNX__
 #  include <sys/socket.h>
 # endif
+# ifdef __OpenBSD__
+#  include <sys/socket.h>
+# endif
```

Comment:
Sounds good.
If we are aren't aware of differences between the BSD.

Do you want to prepare a PR?